### PR TITLE
EZP-25462: Delete an archived in Version tab

### PR DESCRIPTION
**Pull request by yannickroger**
Link: https://jira.ez.no/browse/EZP-25462

## Description
Adds a delete button allowing deleting archived versions.

delete_archived

## Test
Manual and unit tests

### CODE REVIEW

**Comment by yannickroger** [comment]
Ready for review @dpobel @StephaneDiot and others

**Comment by crevillo** [comment]
don't really know if this part of this pull request, but wondering what it is expected to happen when in the "Archived Versions" part you select more than one and click in the `New draft based on Selected version` button. Will it start from the more recent one?

File: Resources/public/js/views/services/plugins/ez-versionsplugin.js [hunk 1]
```diff
@@ -136,19 +136,19 @@ YUI.add('ez-versionsplugin', function (Y) {
...
          * are going to be deleted
          *
-         * @method _deleteDraftConfirm
+         * @method _deleteVersionConfirm
          * @private
-         * @param {EventFacade} e deleteVersionDraft event facade
+         * @param {EventFacade} e deleteVersion event facade
          */
-        _deleteDraftConfirm: function (e) {
+        _deleteVersionConfirm: function (e) {
```
Comment (dpobel) [inline Resources/public/js/views/services/plugins/ez-versionsplugin.js]: I missed it in the previous review but it should probably be called `_confirmDeleteVersion` instead

File: Resources/public/js/views/tabs/ez-locationviewversionstabview.js [hunk 2]
```diff
@@ -202,26 +204,65 @@ YUI.add('ez-locationviewversionstabview', function (Y) {
         },
 
         /**
+         * Returns the selected archived versions as VersionInfo
+         *
+         * @method _getSelectedArchived
+         * @protected
+         * @return {Array} of eZ.VersionInfo
+         */
+        _getSelectedArchived: function () {
```
Comment (dpobel) [inline Resources/public/js/views/tabs/ez-locationviewversionstabview.js]: it's almost the same as `_getSelectedDrafts`, you can probably reorganize a bit the code to use the same method

**Comment by dpobel** [comment]
@crevillo I think this is handled in https://github.com/ezsystems/PlatformUIBundle/pull/647/files#diff-0cd9f2e49a272c207aecf6b781c2e5a3R126

**Comment by yannickroger** [comment]
@crevillo the button should be disabled if you select more than one `New Draft based on selected version`. If you have suicidal tendencies and use dev tool to enable it, it will just not to anything :smile:

**Comment by dpobel** [comment]
besides the 2 inline comments +1 (when Travis agrees on unit tests)

**Comment by crevillo** [comment]
cool :smile: i should read the code before ask :relaxed:

**Pull request merged**